spec/game_spec.rb

```diff
@@ -0,0 +1,116 @@
+require './lib/game.rb'
+
+describe Game  do
+
+    describe 'score' do
+        it 'adds the scores of different rolls, and returns a score' do
+        subject.roll(1)
+        subject.roll(2)
+        expect(subject.score).to eq 3
+        end
+    end
+
+    describe 'end game'  do
+        it 'ends the game after max rolls, and returns the final score' do
+            ((Game::FRAME_MAX * 2) - 1).times{subject.roll(1)}
```

Comment:
This is slightly harder to read - maybe you could rewrite it such as:
```ruby
max_roll_count = ((Game::FRAME_MAX * 2) - 1)
max_roll_count.times { subject.roll(1) }
```

Reply:
Also it looks like this value is used at different places in the tests, maybe you could refactor it into a `let` block at the beginning of the test suite, so it's available everywhere?